apps/examples/testcase/le_tc/kernel/tc_libc_string.c

```diff
@@ -918,6 +919,69 @@ static void tc_libc_string_strtold(void)
 
 	TC_SUCCESS_RESULT();
 }
+#ifdef CONFIG_LIBC_STRERROR
+/**
+* @fn                   :tc_libc_string_strerror_r
+* @brief                :convert the error number to string
+* @Scenario             :put error number to strerror_r and check the return which points the string value
+* API's covered         :
+* Preconditions         :none
+* Postconditions        :none
+* @return               :void
+*/
+static void tc_libc_string_strerror_r(void)
+{
+	FAR char dest_arr[BUFF_SIZE_24];
+	int ret;
+
+	ret = strerror_r(EBUSY, dest_arr, BUFF_SIZE_24);
+	TC_ASSERT_EQ("strerror_r", ret, OK);
+	TC_ASSERT_EQ("strerror_r", strncmp(dest_arr, "Device or resource busy", strlen("Device or resource busy") + 1), 0);
```

Comment:
I think we should use `strncmp(dest_arr, "Device of resource busy", sizeof(dest_arr)` or `strncmp(dest_arr, "Device of resource busy", BUFF_SIZE_24)`

What if `BUFF_SIZE_24` is less than `sizeof("Device or resource busy")`?
`strncmp(dest_arr, "Device or resource busy", strlen("Device or resource busy") + 1)` will make assert.

Reply:
I agree.
I modified BUFF_SIZE_24 to EBUSY_STR_SIZE ( sizeof(EBUSY_STR) ).